### PR TITLE
Adding new templates for google-native

### PR DESCRIPTION
**Pull request by viveklak**

### CODE REVIEW

**Review by infin8x — APPROVED** [review]
These look good! (And I've confirmed that the blog post sample code for `pulumi new` in each language matches your template names)

File: google-native-csharp/${PROJECT}.csproj
Comment (mikhailshilkov) [inline google-native-csharp/${PROJECT}.csproj:10]:
```suggestion
    <PackageReference Include="Pulumi.GoogleNative" Version="0.*" />
```
I assume we'll publish 0.1.0 before merging

File: google-native-go/go.mod
Comment (mikhailshilkov) [inline google-native-go/go.mod:7]: Reminder to update this before merging

**Review by mikhailshilkov — APPROVED** [review]
LGTM modulo updating all reference after the release

**Comment by viveklak** [comment]
> LGTM modulo updating all references after the release

Sounds good. Will hold till updated versions are released, then update.

File: google-native-csharp/${PROJECT}.csproj
Comment (viveklak) [reply google-native-csharp/${PROJECT}.csproj]: Done.

**Pull request merged**
